www/js/app.js

```diff
@@ -386,6 +397,18 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         }, 600);
     });
 
+    //focus #prefix if Home key is pressed
+    function focusPrefixOnHomeKey(key){
+        //check if home key is pressed
+        if(key === 'Home') {
```

Comment:
Minor, but our style uses a space between `if` and parentheses.

Reply:
Done. I will follow this from now on.